.github/workflows/callbacks.yml

```diff
@@ -2,9 +2,9 @@ name: Callbacks Module
 # This workflow runs when a PR is opened that targets code that is part of the callbacks module.
 on:
   pull_request:
-    paths:
-      - '.github/workflows/callbacks.yml'
-      - 'modules/apps/callbacks/**'
+  push:
+    branches:
+      - main
```

Comment:
Will this mean that it only runs on PRs to main. So mergify backports to release branches it will not?

Maybe we just need:

```yaml
on:
  pull_request:
```

Reply:
This also exists right above the push statement. Git diff might not be showing it

Reply:
Yeah I see it, but only having `main` defined in the list of branches means that it will not run on backports, right? 

Or does this config mean that it will run on all pull requests but also on merge commits to `main`? (maybe I misunderstood)

Reply:
I think this should run on every PR. And also run on any push to main (but not pushes to other branches).

Reply:
Cool, thanks for clearing that up! That makes sense I think

Reply:
yup, that's my understanding too, leaving `pull-requests` blank is like a `*` glob pattern